tests/Paysera/WalletApi/MapperTest.php

```diff
@@ -1955,6 +1955,85 @@ public function testEncodeLocationEmptySpot() {
         $this->assertEquals($expectedData, $encodedLocation);
     }
 
+    /**
+     * @dataProvider encodeTimeDataProvider
+     */
+    public function testDecodeTime($data, $expectedException, $expectedResult)
+    {
+        if ($expectedException !== null) {
+            $this->setExpectedException($expectedException);
+        }
+
+        $result = $this->mapper->decodeTime($data);
+
+        if ($expectedResult !== null) {
+            $this->assertInstanceOf(Paysera_WalletApi_Entity_Time::class, $result);
+            $this->assertEquals($expectedResult->getHours(), $result->getHours());
+            $this->assertEquals($expectedResult->getHours(), $result->getHours());
+        }
+    }
+
+    public function encodeTimeDataProvider()
+    {
+        return [
+            'valid data' => [
+                'data' => '9:0',
+                'expectedException' => null,
+                'expectedResult' => new Paysera_WalletApi_Entity_Time(9, 0),
+            ],
+            'wrong format: missed minutes' => [
+                'data' => '9',
+                'expectedException' => InvalidArgumentException::class,
+                'expectedResult' => null
```

Comment:
Here and other places https://github.com/paysera/php-style-guide?tab=readme-ov-file#commas-in-arrays